CHANGES.md

```diff
@@ -1,3 +1,18 @@
+
+
+### Unreleased
+
+#### Added
+
+- **irmin-unix**:
+  - Added a `--hash` parameter to the command-line interface, allowing the hash
+    function to be specified. For BLAKE2b and BLAKE2s, the bit-length may be
+    specified with a trailing slash, as in `--hash=blake2b/16`. (#898, @craigfe)
```

Comment:
Can we also specify the hash parameter in the configuration file?

For instance, for Tezos we would like to do:

```
store: pack
hash: blake2b/32
root: ~/.tezos-node/context
```

Reply:
We can indeed; I have added some detail to CHANGES to make this clear.